embedded-files/fuseml-core-deployment.yaml

```diff
@@ -1,4 +1,37 @@
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-core
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-workloads
+rules:
+- apiGroups:
+  - tekton.dev
+  resources:
+  - pipelines
+  verbs:
+  - create
```

Comment:
Will that be enough? fuseml also creates `TriggerTemplate`, `TriggerBinding` and `EventListener` from Tekton triggers.

There is also a delete/get operation to be implemented (that could also be added later when we do have it).

Reply:
You are right of course